content/docs/2.0/scalers/redis-streams.md

```diff
@@ -89,8 +92,8 @@ spec:
   triggers:
     - type: redis-streams
       metadata:
-        address: REDIS_HOST
-        password: REDIS_PASSWORD # name of the environment variable in the Deployment
+        addressFromEnv: REDIS_HOST
+        passwordFromEnv: REDIS_PASSWORD # name of the environment variable in the Deployment
         stream: my-stream
         consumerGroup: consumer-group-1
         pendingEntriesCount: "10"
```

Comment:
Sample below is not updated: https://github.com/kedacore/keda-docs/pull/241/files#diff-b6608de35c29cd153d0fce47f4357c26R140